src/providers/MeetingProvider/MeetingManager.ts

```diff
@@ -148,6 +148,10 @@ export class MeetingManager implements AudioVideoObserver {
   // Please use `meetingManagerConfig` to use `MeetingManagerConfig` values.
   videoDownlinkBandwidthPolicy: VideoDownlinkBandwidthPolicy | undefined;
 
+  // This variable will be deprecated in favor of `meetingManagerConfig`.
+  // Please use `meetingManagerConfig` to use `MeetingManagerConfig` values.
+  reconnectTimeoutMs: number | undefined;
```

Comment:
We can remove the variable from here, since we wanted to actually use the values from the `meetingManagerConfig` going forward, thats why we added comment on older variables, for newer ones we can just omit it as a separate public variable on the MeetingManager class.